### DISCUSSION TITLE
Changing The texture pattern of the palm in "icubSim"

### DISCUSSION THREAD

**Discussion by LakshithaHKUST** [robotology/community#83] [Q&A] [answered]
Hello,
Is it possible to change the texture pattern in the right or left palm of the simulator?
To add an arbitrary image from a file?
Thanks

**Comment by pattacini** [comment]
Does anybody have any good clue on this?

**Comment by Tobias-Fischer** [comment]
@martz21 might know

**Comment by LakshithaHKUST** [comment, marked as answer]
Hello everyone,

I managed to change the texture pattern `simConfig/models/blueTexture.bmp`, and attach an arbitrary image to the palm. I'll do that for now.

Thanks for the help.
